package.json

```diff
@@ -31,8 +31,15 @@
     "dev": "run-s rebuild:electron dev-runner",
     "dev:web": "node _scripts/dev-runner.js --web",
     "dev-runner": "node _scripts/dev-runner.js",
+    "lint-all": "npm run lint & npm run lint-json & npm run lint-style",
     "lint-fix": "eslint --fix --ext .js,.vue ./",
     "lint": "eslint --ext .js,.vue ./",
+    "lint-json": "eslint --ext .json ./",
+    "lint-style": "stylelint \"**/*.scss\" & stylelint \"**/*.css\"",
```

Comment:
Please avoid using npm, everything else uses yarn. Also the behaviour of `&` is different on Windows compared to in shells on Linux and macOS
On Windows in cmd: `&` makes it run the commands one after the other, regardless of whether the previous one failed or not
in other shells: `&` makes it run the previous commands in the background in parallel and the last one in the foreground

As we already have the `npm-run-all` dependency for other scripts in this `package.json` file, the better solution would be to use either `run-s` to run them in series or `run-p` to run them in parallel (`npm-run-all` will also correctly use yarn if the script was invoked with yarn)